docs/notebooks/Toroidal_current_constraint.ipynb

```diff
@@ -0,0 +1,837 @@
+{
```

Comment:
<p>We probably shouldn't do a vacuum tokamak since it's not really well defined. Maybe do a fixed nonzero current case? And then add 3d shaping, then perturb to zero current?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>